pkg/generator/templates/cloud-init.gardenlinux.template

```diff
@@ -28,8 +38,12 @@ mkdir -p '{{ $unit.DropIns.Path }}'
 
 systemctl daemon-reload
 {{ if .Bootstrap }}
+{{ if isContainerDEnabled .CRI -}}
+systemctl enable containerd && systemctl restart containerd
+{{- else -}}
```

Comment:
Docker always has to be enabled. 

The `cloud-config-execution` script requires docker for the image pre-load & In Gardenlinux both docker and containerd are disabled per default, right?.

I do not like the current dependency on docker to be honest.

Reply:
Done